lang/cs/Org.Apache.REEF.Tang.Tests/ClassHierarchy/TestClassHierarchy.cs

```diff
@@ -427,6 +427,20 @@ public void TestInterface()
             Assert.NotNull(b);
             Assert.NotNull(c);
         }
+
+        // See REEF-1276
+        [Fact]
+        public void TestInjectableClassWithParameterAnnotationThrowsMeaningfulException()
+        {
+            ITang tang = TangFactory.GetTang();
+
+            ICsConfigurationBuilder cb = tang.NewConfigurationBuilder();
+            IConfiguration conf = cb.Build();
+            IInjector i = tang.NewInjector(conf);
```

Comment:
I would replace those three lines to  IInjector i = tang.NewInjector(tang.NewConfigurationBuilder().Build());